slices/dbus.yaml

```
@@ -0,0 +1,35 @@
+package: dbus
+
+essential:
+  - dbus_copyright
+
+slices:
+  bins:
+    essential:
+      - dbus-bin_bins
+      - dbus-daemon_bins
+      - dbus-system-bus-common_config
+      - dbus_config
+      - libc6_libs
+      - libdbus-1-3_libs
+      - libexpat1_libs
+      - libsystemd0_libs
+    contents:
+      /etc/init.d/dbus:
+      /usr/lib/dbus-1.0/dbus-daemon-launch-helper:
```

I wonder if we should omit the `1.0` part.
```suggestion
      /usr/lib/dbus-*/dbus-daemon-launch-helper:
```